upgrade.php

```diff
@@ -244,7 +244,7 @@ function getSqlServer()
     //test if mysql or mariadb is used.
     $sql = 'SELECT VERSION() as version';
     $result = $link->query($sql);
-    if (strpos($result->fetch_array['version'], 'MariaDB') === false) {
+    if (strpos($result->fetch_array()['version'], 'MariaDB') === false) {
```

Comment:
It will exit with error `Parse error: syntax error, unexpected '[', expecting ',' or ';'`

see this test 3v4l: https://3v4l.org/DXgmp

As for php 5.4 minimum version is related to a better utf8 decoding, on php 5.3 UTF8 decoding works, but sometimes gives strange, but not-blocking, results